src/main/resources/com/google/api/codegen/nodejs/test.snip

```diff
@@ -25,6 +25,11 @@
   @else
     var {@apiTest.apiWrapperModuleName} = require('../src/')();
   @end
+  @join testClass : apiTest.testClasses
```

Comment:
Won't this result in many imports of the MockStream if there are many test classes?

I'd imagine it would look more like this:

```
@if apiTest.testClasses.iterator.hasNext
  var MockStream = require('google-gax').MockStream;
@end